tests/test_core/test_alignment.py

```diff
@@ -2571,6 +2571,31 @@ def test_add_from_ref_aln(self):
             ValueError, aln1.add_from_ref_aln, aln2_wrong_refseq
         )  # test wrong_refseq
 
+    def test_annotate_matches_to(self):
+        """annotate_matches_to method should attach
+         annotations correctly to a Sequence object."""
+        seq = Sequence("TTCCACTTCCGCTT", name="x")
+        allow_multiple = True
+        if not isinstance(seq, Sequence):
+            return True
+        pattern = "CCRC"
+        regular_expression = DNA.to_regex(seq=pattern)
+        annot = seq.annotate_matches_to(
+            pattern=regular_expression,
+            annot_type="domain",
+            name="fred",
+            allow_multiple=allow_multiple,
+        )
+        if allow_multiple == True:
```

Comment:
remove the if allow_multiple clause, since it has been removed (it was always True anyway).